test/integration/ordinary/itest_slc.py

```diff
@@ -0,0 +1,225 @@
+import textwrap
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from test.integration.ordinary.test_itde_manager import remove_itde
+
+import pytest
+from docker.models.images import Image as DockerImage
+from exasol_integration_test_docker_environment.lib.docker import ContextDockerClient
+
+from exasol.nb_connector.itde_manager import bring_itde_up
+from exasol.nb_connector.language_container_activation import (
+    open_pyexasol_connection_with_lang_definitions,
+)
+from exasol.nb_connector.secret_store import Secrets
+from exasol.nb_connector.slc.script_language_container import (
+    PipPackageDefinition,
+    ScriptLanguageContainer,
+    constants,
+)
+
+
+@pytest.fixture(scope="module")
+def working_path() -> Path:
+    with TemporaryDirectory() as d:
+        yield Path(d)
+
+
+@pytest.fixture(scope="module")
+def secrets_file(working_path: Path) -> Path:
+    return working_path / "sample_database.db"
+
+
+@pytest.fixture(scope="module")
+def slc_secrets(secrets_file, working_path) -> Secrets:
+    secrets = Secrets(secrets_file, master_password="abc")
+    return secrets
+
+
+@pytest.fixture(scope="module")
+def itde(slc_secrets: Secrets):
+    bring_itde_up(slc_secrets)
+    yield
+    remove_itde()
+
+
+DEFAULT_FLAVOR = "template-Exasol-all-python-3.10"
+
+
+def create_slc(
+    secrets: Secrets,
+    name: str,
+    flavor: str = DEFAULT_FLAVOR,
+) -> ScriptLanguageContainer:
+    return ScriptLanguageContainer.create(secrets, name=name, flavor=flavor)
+
+
+@pytest.fixture(scope="module")
+def sample_slc(slc_secrets: Secrets, working_path: Path) -> ScriptLanguageContainer:
+    return create_slc(slc_secrets, "sample")
+
+
+@pytest.fixture(scope="module")
+def other_slc(slc_secrets: Secrets, working_path: Path) -> ScriptLanguageContainer:
+    """
+    Creates another SLC with a different flavor for verifying operations
+    to be limited to the current SLC only, e.g. removing docker images or
+    working directories.
+    """
+    slc = create_slc(slc_secrets, "other", flavor="template-Exasol-all-r-4")
+    slc.export()
+    slc.deploy()
+    return slc
+
+
+@pytest.fixture
+def custom_packages() -> list[tuple[str, str, str]]:
+    return [("xgboost", "2.0.3", "xgboost"), ("scikit-learn", "1.5.0", "sklearn")]
+
+
+@pytest.mark.dependency(name="export_slc")
+def test_export_slc(sample_slc: ScriptLanguageContainer):
+    sample_slc.export()
+    export_path = sample_slc.workspace.export_path
+    assert export_path.exists()
+    tgz = [f for f in export_path.glob("*.tar.gz")]
+    assert len(tgz) == 1
+    assert tgz[0].is_file()
+    tgz_sum = [f for f in export_path.glob("*.tar.gz.sha512sum")]
+    assert len(tgz_sum) == 1
+    assert tgz_sum[0].is_file()
+
+
+def slc_docker_tag(slc: ScriptLanguageContainer) -> str:
```

Comment:
```suggestion
def slc_docker_tag_prefix(slc: ScriptLanguageContainer) -> str:
```

Reply:
I will fix this manually, as also the usages need to be updated.